pkg/apis/rollouts/v1alpha1/experiment_types.go

```diff
@@ -201,4 +203,6 @@ type ExperimentAnalysisRunStatus struct {
 	Phase AnalysisPhase `json:"phase"`
 	// Message is a message explaining the current status
 	Message string `json:"message,omitempty"`
+	// RequiredForCompletion indicates that experiment should complete after analysis finishes
+	RequiredForCompletion bool `json:"requiredForCompletion,omitempty"`
```

Comment:
I feel we don't need to duplicate `RequiredForCompletion` in the status for this structure. Can't this be inferred from the `ExperimentAnalysisTemplateRef`?

Reply:
Yeah, fair point.